README.md

```diff
@@ -83,6 +83,17 @@ when processing the `html` template:
 error.  This babel transformation can only determine that a template is broken, the
 eslint plugin will tell you which binding is invalid.
 
+- `failOnError` configures whether an error should be thrown when minification failed.
```

Comment:
This is a top-level option like `modules`.  The way it's documented here implies it is an option that gets set inside `htmlMinifier`.  Same goes for `logOnError`.